### PR TITLE
Helper method for mapping ValueContentAbstractValues to PropertySetAbstractValueKinds

### PR DESCRIPTION
**Pull request by dotpaul**
Hopefully this will using PropertySetAnalysis with ValueContentAnalysis a little easier.

### CODE REVIEW

**Comment by dotpaul** [comment]
FYI @LLLXXXCCC @DSIE63 @timhermann @matt-desai

File: src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.cs [hunk 1]
```diff
@@ -191,6 +193,87 @@ public static HazardousUsageEvaluationResult MergeHazardousUsageEvaluationResult
...
+            ValueContentAbstractValue valueContentAbstractValue,
+            Func<object, bool> badLiteralValuePredicate)
+        {
+            Debug.Assert(valueContentAbstractValue != null);
+            Debug.Assert(badLiteralValuePredicate != null);
+
+            switch (valueContentAbstractValue.NonLiteralState)
+            {
+                case ValueContainsNonLiteralState.No:
+                    if (valueContentAbstractValue.LiteralValues.IsEmpty)
```
Comment (LLLXXXCCC) [inline src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.cs]: When the NonLiteralState is No, what case's LiteralValues can be Empty?

Comment (LLLXXXCCC) [reply src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.cs]: I see, the test case TestInstallCertToNullStoreNoDiagnostic.

**Comment by LLLXXXCCC** [comment]
Looks good to me.

**Comment by dotpaul** [comment]
Thank you @LLLXXXCCC !

**Pull request merged**